content/it/infrastructure_as_code.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Infrastructure as Code (IaC - Infrastruttura come Codice)
+status: Feedback Appreciated 
+category: Concetto
+---
+
+## Che cos'è
+Infrastructure as Code è il processo di gestione e provisioning dell’infrastruttura attraverso file di definizione leggibili dalla macchina: tratta la configurazione dell’infrastruttura alla stregua di software di programmazione. Questo sostituisce il modello tradizionale in cui l'infrastruttura come servizio viene creata manualmente, in genere tramite script di shell o altri strumenti di configurazione, ed è elemento fondante del Cloud Computing (quindi essenziale per DevOps).
+
+## Quali problematiche risolve
+La creazione di applicazioni native sul cloud richiede che l'infrastruttura sia a perdere (disposable) e riproducibile. Deve essere inoltre possibile scalarla su richiesta in modo automatizzato e ripetibile, potenzialmente senza intervento umano. Il provisioning manuale, infatti, non può soddisfare i requisiti di reattività e scalabilità delle applicazioni cloud native. Le modifiche manuali all'infrastruttura non sono riproducibili, si scontrano presto con i limiti in termini di scalabilità e introducono il rischio di errori di configurazione. 
```

Comment:
per me 'a perdere' è chiaro, non serve lasciare `disposable`
```suggestion
La creazione di applicazioni native sul cloud richiede che l'infrastruttura sia a perdere e riproducibile. Deve essere inoltre possibile espanderla su richiesta in modo automatizzato e ripetibile, potenzialmente senza intervento umano. Il provisioning manuale, infatti, non può soddisfare i requisiti di reattività e scalabilità delle applicazioni cloud native. Le modifiche manuali all'infrastruttura non sono riproducibili, si scontrano presto con i limiti in termini di scalabilità e introducono il rischio di errori di configurazione. 
```

Reply:
Credo che il concetto di "scalabilità" sia più efficace e rilevante se confrontato con il concetto di "espandibilità" a dirla tutta 😋

Reply:
Capisco. ad ogni modo 'scalare' non è un verbo italiano :D magari vuoi parafrasare per usare il termine 'scalabile'

Reply:
https://docs.microsoft.com/it-it/azure/architecture/best-practices/auto-scaling 
`È possibile scalare un'applicazione in due modi principali` 😏

vabbeh, anche se ho ragione io, mi sta bene un "deve essere scalabile" 😝